pandas/api/extensions/__init__.py

```diff
@@ -11,3 +11,5 @@
 )
 from pandas.core.algorithms import take  # noqa: F401
 from pandas.core.arrays import ExtensionArray, ExtensionScalarOpsMixin  # noqa: F401
+from pandas.core.common import is_bool_indexer  # noqa: F401
```

Comment:
@jbrockmendel ought to move is_bool_indexer to a more usual location, pandas.core.dtypes.common is better

Reply:
i think core.indexers